File: BBIS.Application/DTOs/DonorTestOrder/DonorTestOrderDto.cs
namespace BBIS.Application.DTOs.DonorTestOrder
{
    public class DonorTestOrderDto
    {
        public Guid DonorTestOrderId { get; set; }
        public Guid DonorTransactionId { get; set; }
        public bool TestCompleted { get; set; }
        public string FinalBloodType { get; set; }
        public string BloodRh { get; set; }

        public List<DonorTestOrderTypesDto> TestTypes { get; set; }
    }
}

File: BBIS.Api/ClientApp/src/models/TestOrder/DonorTestOrderDto.ts
import { DonorTestOrderTypesDto } from "./DonorTestOrderTypesDto";

export interface DonorTestOrderDto {
    DonorTestOrderId: Guid;
    DonorTransactionId: Guid;
    TestCompleted: boolean;
    TestTypes: Array<DonorTestOrderTypesDto>;
    FinalBloodType: string;
    BloodRh: string;
}